Dockerfile

```diff
@@ -5,6 +5,12 @@ FROM python:${PYTHON_VER}-slim
 RUN pip install --upgrade pip \
   && pip install poetry
 
+RUN apt-get update && apt-get install -y \
+  gcc \
```

Comment:
Adding `gcc` feels pretty heavyweight for this container - what's it needed for?

Reply:
This comes from psutil, I believe in conjunction with Buildx:

```
#10 5.989         C compiler or Python headers are not installed on this system. Try to run:
#10 5.989         sudo apt-get install gcc python3-dev
#10 5.989         error: command 'gcc' failed: No such file or directory: 'gcc'
#10 5.989         [end of output]
```